recipes/civetweb/all/conanfile.py

```diff
@@ -49,25 +52,15 @@ class CivetwebConan(ConanFile):
         "with_zlib": False,
     }
 
-    generators = "cmake", "cmake_find_package", "cmake_find_package_multi"
-    _cmake = None
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
+    no_copy_source = True
```

Comment:
```suggestion
```

Reply:
without it, build fails before generate method call with:
```sh
ERROR: The folder cache/open-source/.conan/data/civetweb/1.15/_/_/build/a6857e39dd728ca8fee09108efc211b3685e325c/build/Release does not exist and could not be created (Not a directory).
```